packages/react/src/lib/renderComponent.tsx

```diff
@@ -142,7 +142,7 @@ const resolveStyles = (
 
 const renderComponent = <P extends {}>(
   config: RenderConfig<P>,
-  context: ProviderContextPrepared,
+  context?: ProviderContextPrepared,
```

Comment:
I marked `context` as optional to emphasize that it can be `undefined`

Reply:
I understand that this would fix the problem of not throwing when Provider is not defined, but we are still not telling the user, why the Button is not rendered as expected, if there is no Provider :\ It seems like we are silencing the error, instead of explaining it...

Reply:
Actually, we are telling: 
![image](https://user-images.githubusercontent.com/14183168/63442336-38fd2080-c433-11e9-81cc-1a2d9ceed59e.png)

---

But then happens:
![image](https://user-images.githubusercontent.com/14183168/63442359-44504c00-c433-11e9-886e-e2e8c3447eb7.png)

And I want to fix it.

Reply:
Got it, I missed that, thanks for the clarification